### DISCUSSION TITLE
Long term timelapse with Live feed

### DISCUSSION THREAD

**Discussion by AnkurThepade** [gopro/labs#233] [Q&A]
Hello all,

First of all, I am just a content creator but getting hands with the technical aspects to get creative.
Need help in pulling off a project where I am planning to shoot a long-term construction timelapse using the labs.

I have to meet the following conditions - 

- [ ] 1. Should be able to watch the stream from the camera when triggered. NOT the entire time.
- [ ] 2. To create a time-lapse, by shooting photos at the interval from 9 am to 5 pm daily for the next 2 years
- [ ] 3. The data should be able to back up as the camera once setup won't be accessible easily 
- [ ] 4. If a change of plan, I might need to change the timings/interval of the camera remotely

Also considering if I can create a custom trigger that can help me achieve the above conditions.

Thanks in advance!

**Comment by dnewman-gpsw** [comment]
1. I'm not sure you will be able do that. Livestream is not compatibility with any other capture mode (can't take time-lapse photos while LS.) You could use Media Mod, then HDMI to an external LS device. Or you could have a time window each day, when LS is offered, but that would mean no photo capture. Sometimes these things are easier solved using two cameras.
2. The two years part will be the tricky, so for higher reliability, use an external timer, then have Labs have boot commands that run when power is applied.  This is very reliable.  Early testing has shown this to a low cost solution https://www.amazon.com/dp/B083BKBKW5 
3. If you can WiFi access GoPro cloud back is available.
4. This is not available AFAIK. It might be possible via OpenGoPro if that support writing to the camera's SD card, and Labs can support scripts stored on the SD card.

**Comment by Orsbo-byte** [comment]
Hi

I've been using this script with external timers at the power suply for the last 3 weeks with two cameras at a construction site. Works great so far:

!MNAKE=1!MWAKE=2!MBOOT="!Ldaily"!SAVEdaily=mNL>06:00<20:00!5S+!20:00E+!U~!5U

cheers 

Martin

**Reply by DubeAlba** [reply]
When you plug in the power source, do you leave the battery inside the GoPro? Can you explain the command's significance to me? Thank you.
